Rosinstall file missing ceres-solver dependency

Following the instructions exactly on the Read the Docs site causes rosdep install to fail as it can't resolve ceres-solver. I found that using wstool to merge the rosinstall file from cartograper_ros resolved this as that file does include the ceres-solver dependency. I think either updating the instructions to include this step or updating the rosinstall would fix this issue for future users.

I am hitting the same issue. rosdep install can't resolve ceres-solver.

```bash
$ rosdep install --from-paths src --ignore-src --rosdistro=${ROS_DISTRO} -y
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
cartographer: Cannot locate rosdep definition for [ceres-solver]
```

Also catkin build fails.

```bash
CMake Error at CMakeLists.txt:30 (find_package):
  By not providing "FindCeres.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Ceres", but
  CMake did not find one.

  Could not find a package configuration file provided by "Ceres" with any of
  the following names:

    CeresConfig.cmake
    ceres-config.cmake

  Add the installation prefix of "Ceres" to CMAKE_PREFIX_PATH or set
  "Ceres_DIR" to a directory containing one of the above files.  If "Ceres"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!
See also "/home/renwei/workspace/ros/slam/google_cartographer_turtlebot/build_isolated/cartographer/install/CMakeFiles/CMakeOutput.log".
See also "/home/renwei/workspace/ros/slam/google_cartographer_turtlebot/build_isolated/cartographer/install/CMakeFiles/CMakeError.log".
<== Failed to process package 'cartographer': 
  Command '['cmake', '/home/renwei/workspace/ros/slam/google_cartographer_turtlebot/src/cartographer', '-DCMAKE_INSTALL_PREFIX=/home/renwei/workspace/ros/slam/google_cartographer_turtlebot/install_isolated', '-G', 'Ninja']' returned non-zero exit status 1
```

--- a/cartographer_turtlebot.rosinstall
+++ b/cartographer_turtlebot.rosinstall
@@ -1,3 +1,4 @@
 - git: {local-name: cartographer, uri: 'https://github.com/googlecartographer/cartographer.git'}
 - git: {local-name: cartographer_ros, uri: 'https://github.com/googlecartographer/cartographer_ros.git'}
 - git: {local-name: cartographer_turtlebot, uri: 'https://github.com/googlecartographer/cartographer_turtlebot.git'}
+- git: {local-name: ceres-solver, uri: 'https://ceres-solver.googlesource.com/ceres-solver.git', version: '1.12.0rc4'}

